docs/prover/cli/conf-file-api.md

```diff
@@ -0,0 +1,102 @@
+Certora Prover Conf File API
+============================
+
```

Comment:
We should probably mention that the VSCode extension, and also the fact that by convention conf files are stored in `certora/conf` (I think that's right?)

Reply:
Added a section for VS code users